examples/pytorch/graphsage/README.md

```diff
@@ -59,3 +59,9 @@ Train w/ mini-batch sampling for link prediction on OGB-Citation2:
 ```bash
 python3 link_pred.py
 ```
+
+Results:
+```
+Test Accuracy: 0.8600
```

Comment:
I think the metric is `MRR` rather than `accuracy`. Also this seems to be too high if we compare with the results in https://ogb.stanford.edu/docs/leader_linkprop/#ogbl-citation2

Reply:
I'm also curious why we implement a metric function ourselves rather than use the `evaluator` provided by OGB.

Reply:
> I think the metric is MRR rather than accuracy. 

Thanks. I fixed it.

> Also this seems to be too high

Good point. Given only 10 epochs, there is sth wrong I think...

> I'm also curious why we implement a metric function ourselves rather than use the evaluator provided by OGB.

I am not sure why not using ogb evaluator... I definitely agree we should use ogb evaluator for all ogb examples. Not sure if there is any compatibility issue though (like if some users have out-dated pkg, would that be a problem?)

Reply:
@BarclayII Could you explain the reasons for the custom evaluation function?